app/mailers/devise_notify_mailer.rb

```diff
@@ -0,0 +1,49 @@
+class DeviseNotifyMailer < Devise::Mailer
+  require "notifications/client"
+
+  RESET_PASSWORD_TEMPLATE_ID = "4593417c-500f-452c-8111-0f9d311aad0e".freeze
+  SET_PASSWORD_TEMPLATE_ID   = "00cd7163-4213-4596-b4f9-9e72796e0d76".freeze
```

Comment:
Should this be in Constants?

Reply:
Or env vars or smith

Reply:
They could be ENV vars I suppose but they're not expected to ever be used anywhere outside this file and also not sensitive so maybe not a ton of benefit?

Reply:
absolutely fair 👍